src/scenes/Partners/Detail/PartnerDetail.tsx

```diff
@@ -183,96 +184,58 @@ export const PartnerDetail = () => {
                 }
                 empty="Enter PMC Entity Code"
                 loading={!partner}
+                sx={{ border: 'none', background: 'none' }}
               />
             </Grid>
-            <BooleanProperty
-              label="Global Innovations Client"
-              redacted="You do not have permission to view whether this is a Global Innovations Client"
-              data={partner?.globalInnovationsClient}
-              wrap={(node) => <Grid item>{node}</Grid>}
-            />
-          </Grid>
-          <Grid container spacing={3}>
-            <Grid item xs={6} className={classes.cardSection}>
-              <Typography variant="h3">
-                {partner ? 'Partner Types' : <Skeleton width="120px" />}
-              </Typography>
-              <PartnerTypesCard
+          </div>
+          <TabContext value={filters.tab}>
+            <TabList
+              onChange={(_e, tab) => setFilters({ ...filters, tab })}
+              aria-label="partner navigation tabs"
+              className={classes.tabNav}
+            >
+              <Tab
+                label="Partner Profile"
+                value="profile"
+                className={classes.tabStyle}
+              />
+              <Tab label="People" value="people" className={classes.tabStyle} />
+              <Tab
+                label="Projects"
+                value="projects"
+                className={classes.tabStyle}
+              />
+            </TabList>
+            <TabPanel
```

Comment:
All of these panels should be wrapped in a single `<Paper>` to match designs. Or maybe a
```tsx
<Card variant="outlined">
```
Your `tabPanel` class could be put on this `sx: { '> *': {...} }` as well